update_factorio.py

```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
```

Comment:
This could probably just be set to plain `python`; we needn't care what the system has set as default if we can run under both 2 and 3.